org.esbtools.auth.common/pom.xml

```diff
@@ -0,0 +1,34 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.esbtools.auth</groupId>
+    <artifactId>cert-ldap-login-module</artifactId>
+    <version>1.3.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>org.esbtools.auth.common</artifactId>
+
+  <dependencies>
+    <dependency>
+      <groupId>javax.servlet</groupId>
+      <artifactId>servlet-api</artifactId>
+      <version>2.5</version>
+      <scope>provided</scope> <!-- TODO provided? -->
+    </dependency>
```

Comment:
Should this be provided? I left it so containers could provide their own. For example, EAP (for whatever reason) provides it's own version of the apis.

Reply:
The javax.servlet, commons-lang, and guava can be marked provided, the unboundid one cannot.